src/dawn/Optimizer/Stencil.h

```diff
@@ -51,6 +51,7 @@ class Stencil {
     bool IsTemporary;
     std::string Name;
     int AccessID;
+    Array3i Dimensions;
```

Comment:
can u add a comment explaining what is the convention here  [question mark]
should it be 1 0 0 for 1D i dimension [question mark]

Reply:
done